mlflow/models/utils.py

```diff
@@ -264,3 +263,17 @@ def plot_lines(data_series, xlabel, ylabel, legend_loc=None, line_kwargs=None):
     ax.set(xlabel=xlabel, ylabel=ylabel)
 
     return fig, ax
+
+
+def create_wheeled_model(artifact_path, model_uri):
```

Comment:
Currently, `mlflow.models.utils` doesn't show up in the MLflow docs. Can we import it / add it here in the `mlflow.models` module, which is documented? https://github.com/mlflow/mlflow/blob/8098785831a3414bed62ec3239b60a6a500f4dbd/mlflow/models/__init__.py#L43